docs/topics/jvm/jvm-spring-boot-add-data-class.md

```diff
@@ -0,0 +1,154 @@
+[//]: # (title: Add a data class to Spring Boot project)
+[//]: # (description: Add a Kotlin data class to Spring Boot project.)
+
+<microformat>
+    <p>This is the second part of the <strong>Getting started with Spring Boot and Kotlin</strong> tutorial. Before proceeding, make sure you've completed previous steps:</p><br/>
+    <p><img src="icon-1-done.svg" width="20" alt="First step"/> <a href="jvm-create-project-with-spring-boot.md">Create a Spring Boot project with Kotlin</a><br/><img src="icon-2.svg" width="20" alt="Second step"/> <strong>Add a data class to the Spring Boot project</strong><br/><img src="icon-3-todo.svg" width="20" alt="Third step"/> Add database support for Spring Boot project<br/><img src="icon-4-todo.svg" width="20" alt="Fourth step"/> Use Spring Data CrudRepository for database access</p>
+</microformat>
+
+In this part of the tutorial, you'll add some more functionality to the application and discover more Kotlin language features, such as data classes.
+It requires changing the `MessageController` class to respond with a JSON document containing a collection of serialized objects.
+
+## Update your application
+
+1. In the `DemoApplication.kt` file, create a `Message` data class with two properties: `id` and `text`:
+
+    ```kotlin
+    data class Message(val id: String?, val text: String)
+    ```
+
+   `Message` class will be used for data transfer: a list of serialized `Message` objects will make up the JSON document that the controller is going to respond to the browser request.
+
+   <deflist collapsible="true">
+       <def title="Data classes – data class Message">
+          <p>The main purpose of <a href="data-classes.md">data classes</a> in Kotlin is to hold data. Such classes are marked with <code>data</code> keyword and some standard functionality and some utility functions are often mechanically derivable from the class structure.</p>
```

Comment:
I'm not sure that the wording "often mechanically derivable" is correct when we're speaking about data classes. 
Maybe:
some standard functionality and some utility functions are accessible automatically because of a data class structure